### DISCUSSION TITLE
How to disable hotkey?

### DISCUSSION THREAD

**Discussion by ZeronoFreya** [wailsapp/wails#2174] [Q&A]
# browser is edge.

```
System
------
OS:             Windows 10 Home
Version:        2009 (Build: 22000)
ID:             21H2
Go Version:     go1.19.1
Platform:       windows
Architecture:   amd64

Wails
------
Version:        v2.1.0

Dependency      Package Name    Status          Version
----------      ------------    ------          -------
WebView2        N/A             Installed       107.0.1418.62
npm             N/A             Installed       8.19.2
*upx            N/A             Available
*nsis           N/A             Available

* - Optional Dependency

Diagnosis
---------
Your system is ready for Wails development!
Optional package(s) installation details:
  - upx : Available at https://upx.github.io/
  - nsis : Available at https://nsis.sourceforge.io/Download

 ```

Use edge to open edge://extensions/shortcuts，many plugins use shortcut keys to activate extensions and cannot be closed.
These shortcut keys will affect the compiled wails program, and the shortcut keys with the same name set by js will be invalid.
I don't know if this is a bug, but it's something I can't fix.

**Comment by leaanthony** [comment]
Are you saying you want to override a default Edge shortcut (like ctrl-p for print) but you can't? Do you have a specific example?

**Reply by ZeronoFreya** [reply]
For example https://github.com/EdgeTranslate/EdgeTranslate/wiki, The shortcut key for this plugin is Alt+Q，
```js
import hotkeys from "hotkeys-js";
hotkeys('alt+q', function(event, handler){
  // Prevent the default refresh event under WINDOWS system
  event.preventDefault()
  alert('you pressed Alt Q!')
});
```
The shortcut keys set by hotkeys-js are invalid

**Comment by SerenaZzhou** [comment]
I've received your letter, thank you。
